packages/admin/src/targetGroups/addContacts/AddContactsGridSelect.tsx

```diff
@@ -95,7 +95,7 @@ const useSubmitMutation = (id: string) => {
     const [addContactsToTargetGroup] = useMutation<GQLAddBrevoContactsToTargetGroupMutation, GQLAddBrevoContactsToTargetGroupMutationVariables>(
         addBrevoContactsToTargetGroupMutation,
         {
-            refetchQueries: [namedOperations.Query.ManuallyAssignedBrevoContactsGrid, targetGroupFormNamedOperations.Query.TargetGroupForm],
+            refetchQueries: [namedOperations.Query.AssignedBrevoContactsGrid, targetGroupFormNamedOperations.Query.TargetGroupForm],
```

Comment:
Does the allAssignedBrevoContactsQuery also need to be refetched here? So the grid updates when the assignment changes?

Reply:
Good point! 
But I tested it, and the grid already updates, although I can't explain why...

Reply:
If it works, it works 🤷🏻‍♀️  👍🏻